prog/dftb+/prg_dftb/mainio.F90

```diff
@@ -9,7 +9,15 @@
 
 !> Various I/O routines for the main program.
 module mainio
-  use io
```

Comment:
Both doxygen and ford fail for this file, but work for the main version, not sure why.

Reply:
But the files should be preprocessed with Fypp before they reach the documentation system. I'll check, whether that happens at all.